src/styles/button.scss

```
@@ -1,6 +1,6 @@
 $button-family: $family-alternative;
 
-@import "../../node_modules/bulma/sass/elements/button.sass";
+@import "bulma/sass/elements/button.sass";
```

The tests are failing because Jest cannot locate Bulma, but if you write it as follows, tests will pass.

```suggestion
@import "node_modules/bulma/sass/elements/button.sass";
```

---

@hugosolar this is the solution, but I'm not sure if it fixes the original problem that @brenoferreira was aiming to solve.

---

Fixed it in 460aaca